### PR TITLE
Release v0.8.0

### PR DESCRIPTION
**Pull request by nsheff**
- Improves mkabs function and is_url function

### CODE REVIEW

**Comment by jayvdb** [comment]
It would be nice if you could include #42

File: ubiquerg/paths.py [hunk 1]
```diff
@@ -78,10 +81,10 @@ def parse_registry_path(
     return parsed_identifier
 
 
-def mkabs(path, reldir=None):
+def mkabs(path: str, reldir: str = None) -> str:
```
Comment (khoroshevskyi) [inline ubiquerg/paths.py:84]: nice

File: ubiquerg/paths.py [hunk 2]
```diff
@@ -94,10 +97,19 @@ def mkabs(path, reldir=None):
     def xpand(path):
         return os.path.expandvars(os.path.expanduser(path))
 
+    if path is None:
```
Comment (khoroshevskyi) [inline ubiquerg/paths.py:100]: Why do we want to return None if path is None?

**Review by khoroshevskyi — APPROVED** [review]
Small question, otherwise approved

**Pull request merged**